kv/txn_coord_sender.go

```diff
@@ -504,8 +503,8 @@ func (tc *TxnCoordSender) sendOne(ctx context.Context, call proto.Call) {
 		// without starting a new call, which is hard to trace. Plus, the
 		// below depends on default configuration.
 		tmpDB, err := client.Open(
-			fmt.Sprintf("//%s?priority=%d",
-				call.Args.Header().User, call.Args.Header().GetUserPriority()),
+			fmt.Sprintf("//?priority=%d",
```

Comment:
I think that should have been `%s@` (i.e. `user@`). I think this is still needed to pass along the user correctly.